dev-php/pecl-ast/pecl-ast-1.0.5.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_PHP="php7-1 php7-2 php7-3 php7-4"
+inherit php-ext-pecl-r3
+
+DESCRIPTION="A succinct parallel concurrency API for PHP7"
+LICENSE="MIT"
```

Comment:
Make sure this is right. Looked to me like it's ISC and not MIT.

Reply:
It most closely resembles https://opensource.org/licenses/BSD-3-Clause which I believe is BSD in portage.